float32 x_error 
#gives the difference between turtle position and feed forward expected position in x
float32 y_error
#gives the same in y
float32 theta_error
#error in theta
